build.xml

```diff
@@ -50,6 +50,11 @@
             <arg value="phpunit/php-invoker:^2.0"/>
         </exec>
 
+        <exec executable="${basedir}/build/tools/composer" taskname="composer">
```

Comment:
@theofidry Is php-scoper available as a PHAR? If so, @kambo-1st, I would prefer to have that PHAR in `build/tools` and used here.

Reply:
It should. There was a problem with the last version so it didn't get a PHAR but that shouldn't happen anymore.

However I would recommend to switch to a PHAR at the last moment since it makes it much harder to debug/fix in the meantime.

Reply:
Sure, using Composer for the moment is fine until we are sure that 1) php-scoper can handle PHPUnit and 2) PHPUnit uses php-scoper correctly.

Reply:
I also prefer PHAR. But as @theofidry  mentioned it wasn't available for latest version. When it will be available, i will use it.